[2021-10-16 13:05:46,192] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:26:00+00:00 [queued]>
[2021-10-16 13:05:46,317] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:26:00+00:00 [queued]>
[2021-10-16 13:05:46,318] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:46,318] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:46,318] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:46,435] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_2> on 2021-10-16 12:26:00+00:00
[2021-10-16 13:05:46,470] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_2', 'scheduled__2021-10-16T12:26:00+00:00', '--job-id', '486', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpedq0q48m', '--error-file', '/tmp/tmps2rr9n7q']
[2021-10-16 13:05:46,471] {standard_task_runner.py:77} INFO - Job 486: Subtask get_gross_year_2
[2021-10-16 13:05:46,446] {standard_task_runner.py:52} INFO - Started process 942 to run task
[2021-10-16 13:05:46,888] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:26:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:47,129] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T12:26:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T12:26:00+00:00
[2021-10-16 13:05:47,174] {python.py:152} INFO - Done. Returned value was: ['$32.40 ', '2009\n']
[2021-10-16 13:05:47,448] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_2, execution_date=20211016T122600, start_date=20211016T130546, end_date=20211016T130547
[2021-10-16 13:05:47,565] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:05:47,736] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
